addons/web/static/tests/components/dropdown_menu_tests.js

```diff
@@ -70,6 +71,51 @@ odoo.define('web.dropdown_menu_tests', function (require) {
             dropdown.destroy();
         });
 
+        QUnit.test('only one dropdown rendering at same time (owl vs bootstrap dropdown)', async function (assert) {
+            assert.expect(6);
+            const form = await testUtils.createView({
```

Comment:
Isn't it a bit heavy to spawn an entire form view here? Perhaps you could get away with creating and appending a node on the fly, set its innerHTML and remove it at the end of the test?